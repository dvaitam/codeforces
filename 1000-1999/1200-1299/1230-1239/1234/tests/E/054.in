9 8
4 8 2 3 4 2 5 3
