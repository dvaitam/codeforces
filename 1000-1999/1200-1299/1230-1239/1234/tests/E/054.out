19 23 21 21 17 17 17 15 19
